### PR TITLE
Afrikaans to English

### PR DESCRIPTION
**Pull request by Michael-Beukman**
An Afrikaans to English baseline translation model, trained for 23 epochs, using JW300.

The final results were:
 BLEU dev | BLEU test
 --- | ---
 51.47 | 57.22

### CODE REVIEW

File: benchmarks/af-en/jw300-baseline/README.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+Example 3
+```
+Source:     HOEKOM MOET ONS GEESTELIKE DOELWITTE STEL ?
+Reference:  WHY SET SPIRITUAL GOALS ?
+Hypothesis: HOW DO WE SHOULD WE STECTS DOELVITS ?
+```
+
+
+## Qualitative Analysis
```
Comment (juliakreutzer) [inline benchmarks/af-en/jw300-baseline/README.md:48]: Thank you so much for giving these insights!

File: benchmarks/af-en/jw300-baseline/README.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+Source: `Hy het selfs die woorde “ u seun Dawid ” met verwysing na homself gebruik`
+
+Hypothesis: `He even used the words “ your son David ” with reference to himself`
+
+Reference: `He even referred to himself as “ your son David , ”`
+
+The model's translation above is very literal, almost word for word, whereas the reference is more holistically translated, even though both mean the same thing.
+
+## Observations
+The dataset was quite large, comparably larger than other African languages, and this made training quite slow, and 20 epochs took about 32 hours to train.
```
Comment (juliakreutzer) [inline benchmarks/af-en/jw300-baseline/README.md:63]: Thanks for the patience :)

**Review by juliakreutzer — APPROVED** [review]
Thank you @Michael-Beukman, looks great. Could you please also add the results to https://github.com/masakhane-io/masakhane-mt/blob/master/language_pairs.md? Thanks!

File: benchmarks/af-en/jw300-baseline/README.md (hunk 1 above)
Comment (Michael-Beukman) [reply benchmarks/af-en/jw300-baseline/README.md:48]: Thanks @juliakreutzer. I added this (and the 2 other) language pairs in this pull request: https://github.com/masakhane-io/masakhane-mt/pull/204

Comment (juliakreutzer) [reply benchmarks/af-en/jw300-baseline/README.md:48]: Thank you!

**Pull request merged**